### ISSUE TITLE
Adjust messaging buffer sizes

### ISSUE THREAD

**Issue by lbradstreet** [onyx-platform/onyx#308]
These include:
inbound-buffer-size
completion-buffer-size
release-ch-buffer-size

These are all too big, however I'm not sure what the right size should be. I believe if they're reduced, it's now ok to block on >!! rather than drop, as the cluster should auto-engage backpressure which _should_ allow the queues to drain.

I'd like some advice about the blocking part, as I'm not sure if this could cause deadlocks in go-loops in places. I know that's a thing.

**Comment by lbradstreet** [comment]
cc @MichaelDrogalis

**Comment by MichaelDrogalis** [comment]
My first thought is that shutting everything down may grind to a halt. If the channel are full and we start tearing down a task, threads might not be able to spin down. If we shut everything down in the correct order, this shouldn't be an issue given that calling `close!` on a channel unblocks `put!` calls.

Another thing that comes to mind is what happens if a thread that is taking from a channel crashes. If we fail to recover properly, the `put!` will never unblock.

I think those are the only two cases I can come up with for now - but any problems we find are more likely bugs than flaws with using bounded, blocking channels.

**Comment by lbradstreet** [comment]
No longer relevant in 0.10.
